spring-batch-core/src/test/java/org/springframework/batch/core/JobExecutionTests.java

```diff
@@ -250,4 +258,9 @@ public void testFailureExceptions() {
 		assertTrue(allExceptions.contains(exception));
 		assertTrue(allExceptions.contains(stepException1));
 	}
+
+	private void simulatingARunningExecution() {
```

Comment:
We can inline this method then remove it. This:

```java
@Test
public void testIsRunning() {
   execution.setStartTime(new Date());
   assertTrue(execution.isRunning());
   execution.setEndTime(new Date(100L));
   assertFalse(execution.isRunning());
}
```

is IMO more coherent than this:

```java
@Test
public void testIsRunning() {
   simulatingARunningExecution();
   assertTrue(execution.isRunning());
   execution.setEndTime(new Date(100L));
   assertFalse(execution.isRunning());
}
```